admin/controller/altapay/templates/admin/controller/altapay.twig.php

```diff
@@ -181,6 +184,10 @@ public function index()
             $data['payment_altapay_{key}_currency_id'] = $this->config->get('payment_Altapay_{key}_currency_id');
         }
 
+        if (!empty($data['payment_altapay_{key}_secret'])) {
+            $data['payment_altapay_{key}_secret'] = '*****';
+        }
+
```

Comment:
If the merchant changes another parameter of the account profile, would the password then be reset to `*****`, effectively disabling the user's access, and potentially, letting an attacker login using `*****` ?